lib/axe/api/a11y_check.rb

```diff
@@ -37,11 +37,18 @@ def call(page)
       private
 
       def audit(page)
-        yield page.execute_async_script "#{METHOD_NAME}.apply(#{Core::JS_NAME}, arguments)", @context.to_json, @options.to_json
+        yield page.execute_async_script "#{METHOD_NAME}.apply(#{Core::JS_NAME}, arguments)", *js_args
+      end
+
+      def js_args
+        [ @context, @options ]
+          .reject{ |arg| arg.empty? }
```

Comment:
I know there have been some changes to the JS lib that might accept empty-object params now. It's _possible_ (I'm unsure) that the removal of empty objects is no longer necessary. Do you know?